app/Model/AppModel.php

```diff
@@ -1533,6 +1533,48 @@ public function validateEnumeration($coId, $attribute, $value) {
     
     return true;
   }
+
+  /**
+   * Try to normalize a given Attribute Enumeration
+   *
+   * @since  COmanage Registry v4.0.0
+   * @param  integer $coId      CO ID
+   * @param  string  $attribute Attribute, in Model.attribute form
+   * @param  string  $value     Value to normalize
+   * @return string             The normalized value
+   */
+
+  public function normalizeEnumeration($coId, $attribute, $value) {
+    // First, see if there is an enumeration defined for $coId + $attribute.
+
+    $args = array();
+    $args['conditions']['AttributeEnumeration.co_id'] = $coId;
+    $args['conditions']['AttributeEnumeration.attribute'] = $attribute;
+    $args['conditions']['AttributeEnumeration.status'] = SuspendableStatusEnum::Active;
+    $args['contain'] = false;
+
+    $AttributeEnumeration = ClassRegistry::init('AttributeEnumeration');
+    $attrEnum = $AttributeEnumeration->find('first', $args);
+
+    if(empty($attrEnum['AttributeEnumeration']['dictionary_id'])) {
+
+      $this->Behaviors->load('Normalization');
```

Comment:
Sorry, for #3 I just meant to remove the part of the comment about "other values". The rest of the comment was useful :)

```suggestion
    if(empty($attrEnum['AttributeEnumeration']['dictionary_id'])) {
      // If there is no dictionary attached to the Attribute Enumeration
      // configuration then load the normalizer
      $this->Behaviors->load('Normalization');
```

Reply:
Done